aiodocker/docker.py

```diff
@@ -86,8 +86,8 @@ def __init__(self,
         self.images = DockerImages(self)
         self.volumes = DockerVolumes(self)
 
-    async def close(self):
```

Comment:
@asvetlov sorry it's a work in progress, don't waste your time looking at it at the moment. Thanks for the remark though I was totally mistaking.